azure-pipelines/end-to-end-tests-dir/registries.ps1

```diff
@@ -201,7 +201,7 @@ try
 
     Run-Vcpkg install @builtinRegistryArgs '--feature-flags=registries,manifests'
```

Comment:
We also need to add a test that actually ensures the reference is put into the lock json.